gpudirect-tcpx/tcpx-metrics-server.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: tcpx-metrics
+  namespace: kube-system
+  labels:
+    k8s-app: tcpx-metrics
+spec:
+  selector:
+    matchLabels:
+      k8s-app: tcpx-metrics
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: tcpx-metrics
+        k8s-app: tcpx-metrics
+    spec:
+      priorityClassName: system-node-critical
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: cloud.google.com/gke-accelerator
+                    operator: In
+                    values:
+                      - nvidia-h100-80gb
+      tolerations:
+        - operator: "Exists"
+      hostNetwork: true
+      containers:
+        - image: us-docker.pkg.dev/gce-ai-infra/gpudirect-tcpx/tcpx-metrics:latest
+          name: tcpx-metrics
+          resources:
+            requests:
+              cpu: 150m
+          securityContext:
+            privileged: true
+            capabilities:
+              add:
+                - NET_ADMIN
```

Comment:
Why is it both privileged and CAP_NET_ADMIN?

Reply:
Thanks for catching this, removed.